news.css

```diff
@@ -0,0 +1,35 @@
+body {
+margin:0;
+padding:0;
+background-color: #005689;
+max-width: 1000px;
+margin: auto;
+font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+text-align: center;
+text-decoration: none;
```

Comment:
Do you have to specify 'none', or could you leave out the line for this property?